### ISSUE TITLE
Web app version

### ISSUE THREAD

**Issue by JustHumanz** [xinntao/Real-ESRGAN#90]
did Real-ESRGAN have any plan for make it web app version?

**Comment by xinntao** [comment]
Thanks.

Currently, there is no plan for make it web app version. 
You can try 
- colab: https://colab.research.google.com/drive/1k2Zod6kSHEvraybHl50Lys0LerhyTMCo?usp=sharing
- executable files: https://github.com/xinntao/Real-ESRGAN#Portable-executable-files
-  Huggingface: https://huggingface.co/spaces/akhaliq/Real-ESRGAN
- etc

**Comment by JustHumanz** [comment]
cool,but i think it's will be great if `Real-ESRGAN` have some rest-api with web interface so other user with non technical experience can try it 
i have been try Huggingface but it's very laggy and we can't run it on local mechine for development (or i just don't know to deploy it on local ?) and not scalable

anyway great job create this awesome tools

**Comment by medram** [comment]
It's not about the web version, it's about the GPUs power behind it, in other words, it's costing a little much,
instead, there are much more areas to invest efforts/time into, like improving the AI model.

**Comment by JustHumanz** [comment]
yeah i know about it,web version will not improve the performance

**Comment by JustHumanz** [comment]
improving the AI model was the main focus but creating the interface for end-user was also important too

**Comment by medram** [comment]
I'm a web developer, I could make a nice user interface for this AI/model for online testing, just I need a server/hosting with GPU of course.

**Comment by thusinh1969** [comment]
Very simple to deploy a Flask API folks to serve this model on pyTorch.
Steve

**Comment by letterl** [comment]
> 部署 Flask API 人员以在 pyTorch 上为该模型提供服务非常简单。 史蒂夫



> 部署 Flask API 人员以在 pyTorch 上为该模型提供服务非常简单。 史蒂夫



> Very simple to deploy a Flask API folks to serve this model on pyTorch. Steve

Hello, can you provide simple examples or guidelines? thank you

**Comment by letterl** [comment]
> Very simple to deploy a Flask API folks to serve this model on pyTorch. Steve

Hello, can you provide simple examples or guidelines? thank you

**Comment by twomol** [comment]
> cool,but i think it's will be great if `Real-ESRGAN` have some rest-api with web interface so other user with non technical experience can try it i have been try Huggingface but it's very laggy and we can't run it on local mechine for development (or i just don't know to deploy it on local ?) and not scalable
> 
> anyway great job create this awesome tools

Is your purpose  to learn/try the model or wanna making some commercial service of this model?
